apps/perf-test/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
Do you need this file?

Reply:
I think so, its where the App is rendered via `perf-test.js`. https://github.com/OfficeDev/office-ui-fabric-react/pull/8342/files#diff-f3e56f13bc159ea561c249d3a7cf897fR9

Reply:
Got it. In that case, maybe the title should be updated to something besides "Todo App" :)